MekHQ/src/mekhq/campaign/stratcon/StratconRulesManager.java

```diff
@@ -447,9 +451,6 @@ private static void assignAppropriateExtraForceToScenario(StratconScenario scena
      *
      */
     public static void commitPrimaryForces(Campaign campaign, AtBContract contract, StratconScenario scenario, StratconTrackState trackState) {
```

Comment:
keeping contract for future use?

Reply:
Even if I was, I can't imagine what it'd be at this point, so I'm going to remove it.